app/views/dashboard/protocols/form/_protocol_form.html.haml

```diff
@@ -26,7 +26,7 @@
     - if @protocol.type.capitalize == "Study"
       .edit-study-view.container-fluid
         .row.user-edit-protocol-view
-          = render partial: 'dashboard/protocols/form/study_fields', locals: { form: form, protocol: protocol, admin: admin, permission_to_edit: permission_to_edit }
+          = render partial: 'dashboard/protocols/form/study_fields', locals: { form: form, protocol: protocol, admin: admin, permission_to_edit: permission_to_edit, action_name: action_name }
```

Comment:
I believe you already have access to action_name, so no need to pass it in. Should be provided (http://apidock.com/rails/v4.2.1/ActionDispatch/Routing/Mapper/Scoping/Scope/action_name)